.devcontainer/devcontainer.json

```diff
@@ -15,7 +15,10 @@
 				"twxs.cmake",
 				"ms-vscode.cmake-tools",
 				"llvm-vs-code-extensions.vscode-clangd",
-				"ms-azuretools.vscode-docker"
+				"ms-azuretools.vscode-docker",
+				"mhutchie.git-graph",
+				"eamodio.gitlens",
```

Comment:
I feel that gitlens might be too bloated with features and invasive in the overall development experience in vscode. What features of gitlens that are not handled by gitgraph or native vscode git tools are required for a good developer experience for this project? I wanted to add some of the other extensions too but decided to leave them out and have only what's very tightly connected to developing a cmake project like ours, but I will be glad to hear your points.